x-pack/plugins/enterprise_search/server/integrations.ts

```diff
@@ -280,7 +280,7 @@ const workplaceSearchIntegrations: WorkplaceSearchIntegration[] = [
       }
     ),
     categories: ['enterprise_search', 'file_storage', 'microsoft_365'],
-    uiInternalPath: '/app/enterprise_search/workplace_search/sources/add/sharepoint_server',
+    uiInternalPath: '/app/enterprise_search/workplace_search/sources/add/share_point_server/custom',
```

Comment:
`sharepoint_server` -> `share_point_server` aligns with https://github.com/byronhulcher/kibana/blob/73924b9e8cba21c01fd9c97ec9d22cfef75c0bf5/x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/source_data.tsx#L608 now that we're doing a lookup for source data based on base service type